bokehjs/src/coffee/tool/inspectors/inspect_tool.coffee

```diff
@@ -28,7 +28,8 @@ class InspectTool extends Tool.Model
   event_type: "move"
 
   nonserializable_attribute_names: () ->
-    super().concat(['event_type', 'inner_only'])
+    attrs = _.without(super(), 'active')
+    return attrs.concat(['event_type', 'inner_only'])
```

Comment:
That (`Override`) seems better to me. 

FWIW the `always_active` was originally the means to enable pan and wheel zoom tools on at the same time, before events were more carefully handled separately by gesture type. It can definitely go now.